src/adaptation/icap/ModXact.cc

```diff
@@ -2063,7 +2063,9 @@ void Adaptation::Icap::ModXactLauncher::updateHistory(bool doStart)
 }
 
 bool Adaptation::Icap::TrailerParser::parse(const char *buf, int len, int atEnd, Http::StatusCode *error) {
-    const int parsed = trailer.parse(buf, len, atEnd, hdr_sz);
+    Http::ContentLengthInterpreter interpreter;
+    // TODO: if possible, configure the interpreter with status code
```

Comment:
Yes, that ought to be possible -- the trailer comes some time after the ICAP response which does have a status code.